wandb/wandb_torch.py

```diff
@@ -228,7 +228,7 @@ def log_tensor_stats(self, tensor, name):
         # in min()/max() above. Swap here to prevent a runtime error.
         if tmin > tmax:
             tmin, tmax = tmax, tmin
-        tensor = flat.histc(bins=self._num_bins, min=tmin, max=tmax)
+        tensor = flat.cpu().histc(bins=self._num_bins, min=tmin, max=tmax)
         tensor = tensor.cpu().clone().detach()
```

Comment:
you could probably remove the `cpu` here (just in case, it should be a no-op, but better safe), if we do it for `flat`.
The only issue to keep in mind is this is will hurt performance, we will be communicating a larger tensor (might be a problem when we have a lot of tensors) and performing computation on host instead of the accelerator.